starter-code/index.html

```diff
@@ -9,6 +9,30 @@
   <script type="text/javascript" src="./js/index.js"></script>
   <title>Ironhack cart</title>
 </head>
-<body>
+<body onload="getPriceByProduct()">
```

Comment:
This is not a common use. 
In your js you have `window.onload = function() { ... }` that does the same as `<body onload="getPriceByProduct()">`